src/main/webapp/app/instructor-dashboard/instructor-dashboard-result-dialog.controller.js

```diff
@@ -42,9 +53,23 @@
         }
 
         function onSaveSuccess(result) {
-            $scope.$emit('artemisApp:resultUpdate', result);
-            $uibModalInstance.close(result);
-            vm.isSaving = false;
+            if(vm.feedbacks.length > 0) {
```

Comment:
It seems that you send each feedback in a single REST request. Could you not add all feedback to the manual result and then upload this result in one REST request?

Reply:
Done, see above.